.travis.yml

```diff
@@ -0,0 +1,26 @@
+jdk:
+  - oraclejdk8 # openJDK crashes sometimes; tensorframes is compiled w.r.t JDK8
+
+sudo: required
+
+dist: trusty
+
+cache:
+  directories:
+    - $HOME/.ivy2
+    - $HOME/.sbt/launchers/
+    - $HOME/.cache/spark-versions
+
+env:
+  matrix:
+    - SCALA_VERSION=2.11.8 SPARK_VERSION=2.1.1 SPARK_BUILD="spark-2.1.1-bin-hadoop2.7" SPARK_BUILD_URL="http://d3kbcqa49mib13.cloudfront.net/spark-2.1.1-bin-hadoop2.7.tgz"
```

Comment:
Only running scala tests for this PR. We will merge this file with the master travis file once this PR is checked in.